src/main/java/org/fundacionjala/fc/client/Database.java

```diff
@@ -0,0 +1,53 @@
+package org.fundacionjala.fc.client;
+
+import org.fundacionjala.fc.config.Environment;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Database {
+
+    private static final String DB_URL = Environment.getInstance().getDbMysqlUrlConection();
+    private static final String DB_USERNAME = Environment.getInstance().getDbUserName();
+    private static final String DB_PWD = Environment.getInstance().getDbPassword();
+    private Connection dbConnection;
+
+    /**
+     * Constructor for the Database.
+     *
+     * @throws SQLException
+     */
+    public Database() throws SQLException {
+        dbConnection = DriverManager.getConnection(DB_URL, DB_USERNAME, DB_PWD);
+    }
+
+    /**
+     * Makes a query.
+     *
+     * @param query
+     * @return query result as Map<String, String>
+     * @throws SQLException
+     */
+    public Map<String, String> query(final String query) throws SQLException {
```

Comment:
Please consider converting this class into a Singleton class and Rename the class to DatabaseClient.

Create the following class variables:
```java
private static final Logger LOGGER = LogManager.getLogger(DatabaseClient.class);
private static DatabaseClient instance;
private static Connection dbConnection;
```

Define a private constructor that connects to the database:
```java
    private DatabaseClient() {
        try {
            dbConnection = DriverManager.getConnection(DB_URL, DB_USERNAME, DB_PWD);
        } catch (SQLException e) {
            LOGGER.error("Cannot connect to the database.");
            LOGGER.error(e.getMessage());
        }
    }
```

Create the corresponding getInstance method for singleton classes.
```java
    public static DatabaseClient getInstance() {
        if (instance == null) {
            instance = new DatabaseClient();
        }
        return instance;
    }
```

Rename the runQuery method and define a try catch clausule:
```java
    public static Map<String, String> runQuery(final String query) {
        Map<String, String> results = new HashMap<>();
        try {
            Statement stmt = dbConnection.createStatement();
            ResultSet rs = stmt.executeQuery(query);
            ResultSetMetaData metaData = rs.getMetaData();
            int columns = metaData.getColumnCount();
            while (rs.next()) {
                for (int i = 1; i <= columns; i++) {
                    String key = metaData.getColumnName(i);
                    String value = rs.getString(i);
                    results.put(key, value);
                }
            }
        } catch (SQLException e) {
            LOGGER.error("Cannot connect to the database.");
            LOGGER.error(e.getMessage());
        }

        return results;
    }
```

Implement a close connection method.
```java
    public static void closeConnection() {
        if (dbConnection != null) {
            try {
                dbConnection.close();
            } catch (SQLException e) {
                LOGGER.error("Cannot disconnect to the database.");
                LOGGER.error(e.getMessage());
            }
        }
    }
```